test/controllers/trust.information.controller.spec.ts

```diff
@@ -163,8 +193,9 @@ describe("TRUST INFORMATION controller", () => {
     });
 
     test("catch error when rendering the page", async () => {
-      mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_MOCK);
-      mockGetApplicationData.mockImplementationOnce(() => { throw ERROR; });
+      // mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_MOCK);
```

Comment:
this comment can be removed if not needed

Reply:
removed.